src/dynamic.ts

```diff
@@ -42,15 +42,15 @@ export default class DynamicThreadPool<
   }
 
   protected _chooseWorker (): WorkerWithMessageChannel {
-    let worker: WorkerWithMessageChannel
+    let worker: WorkerWithMessageChannel | undefined
     for (const entry of this.tasks) {
       if (entry[1] === 0) {
         worker = entry[0]
         break
       }
     }
 
-    if (worker) {
+    if (worker != null) {
```

Comment:
I would really prefer to have an helper method for all these null or undefined check. 

A file like utils.ts with: 

```
export function isNullOrUndefined<T>(object: T): boolean {
  return object == null;
}
```

And replacing such checks by:

`!isNullOrUndefined(worker)`
...


Or an Utils class containing only static methods as helpers.

Reply:
I would prefer to disable this eslint rule and use `!object` checks
They are not even smaller and common used, but also they are `falsy` and not `null/undefined` checks
So e.g. for the `!this.filePath` it also checks for empty string

If @pioardi really want something like util functions like `isNullOrUndefined` we should use a library for that. e.g. [isNil](https://github.com/justinlettau/ts-util-is/blob/master/src/index.ts#L181)

But really... I think this is absolutely overkill

Reply:
> I would prefer to disable this eslint rule and use `!object` checks
> They are not even smaller and common used, but also they are `falsy` and not `null/undefined` checks
> So e.g. for the `!this.filePath` it also checks for empty string

If ts-standard is using eslint + TS plugin behind the scene, there should be a way to disable some rule globally.

Reply:
But I would prefer to let decide @pioardi if we should do that, because it would be introducing another config file to the project

Reply:
ts-standard documentation: 

🚫 Please change X rule

This project has no control over the rules implemented, as such this project cannot change any of the rules that have been configured. If you want to discuss the rules, please visit the rules configuration repo eslint-config-standard-with-typescript.

Reply:
Oh I honestly I am not sure on this.
I am more familiar with the original js syntax like 

`if(!this.x)`

Now I understand that with typescript this could not be the best practice, so if we are concerned from that we can just write an util method.

No external libraries please for this tiny thing, this project runs without other 3rd part libraries and unless that introduce a library enable us for something great we should avoid them.

Reply:
Okay we should disable the lint rule for that, I will try...